common/azure-common/src/main/java/com/azure/common/implementation/util/ImplUtils.java

```diff
@@ -3,6 +3,12 @@
 
 package com.azure.common.implementation.util;
 
+import com.azure.common.http.rest.PagedResponse;
```

Comment:
I'm not sure how this PR works - you're doing it into azure-common, which no longer exists?

Reply:
Yup, timing issue, 
Updating the PR